pypopquiz/backends/moviepy.py

```diff
@@ -162,13 +232,27 @@ def add_audio(self, other: 'Moviepy') -> None:  # type: ignore
 
         self.has_audio = True
 
-        if isinstance(other.clip, moviepy.editor.AudioClip):
+        if isinstance(other.clip, med.AudioClip):
             audio = other.clip
         else:
             audio = other.clip.audio
 
         self.clip = self.clip.set_audio(audio)
 
+    def audio_normalize(self) -> None:
```

Comment:
Do you use this somewhere or is it just convenient to have for later?

Reply:
I think it will come in useful later. I added it because I didn't want to forget about it.